src/effects.js

```diff
@@ -631,15 +633,25 @@ jQuery.each( {
 } );
 
 jQuery.timers = [];
+var oldLength = 0;
 jQuery.fx.tick = function() {
 	var timer,
 		i = 0,
 		timers = jQuery.timers;
 
 	fxNow = jQuery.now();
+	oldLength = timers.length;
+	for ( ; i < oldLength; i++ ) {
+		var removedTimers = 0;
 
-	for ( ; i < timers.length; i++ ) {
-		timer = timers[ i ];
+		for ( var j = 0; j < lis.length; j++ ) {
```

Comment:
@gibson042 
![error_bug_1](https://cloud.githubusercontent.com/assets/16049255/22600319/fb1c3454-ea5f-11e6-8183-ca66dacf7921.png)


In every iteration, making the value of the iterator equal to the value of the previous timer that successfully ran + 1.